README.md

```
@@ -180,17 +186,84 @@ main().catch((error) => {
 });
 ```
 
+### Deployment Tracking
+
+Movehat **automatically tracks deployments** per network, similar to hardhat-deploy:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)